opm/parser/eclipse/Deck/Deck.cpp

```diff
@@ -132,6 +132,14 @@ namespace Opm {
         m_dataFile("")
     {}
 
+    Deck::Deck( std::initializer_list< DeckKeyword > ilist ) :
+        Deck( std::vector< DeckKeyword >( ilist.begin(), ilist.end() ) )
+    {}
```

Comment:
Maybe. 

C++11:
> The underlying array is not guaranteed to exist after the lifetime of the original initializer list object has ended. The storage for std::initializer_list is unspecified (i.e. it could be automatic, temporary, or static read-only memory, depending on the situation).

C++14:
> The underlying array is a temporary array (until C++17)a materialized array prvalue (since C++17) of type const T[N], in which each element is copy-initialized (except that narrowing conversions are invalid) from the corresponding element of the original initializer list. The lifetime of the underlying array is the same as any other temporary object, except that initializing an initializer_list object from the array extends the lifetime of the array exactly like binding a reference to a temporary (with the same exceptions, such as for initializing a non-static class member). The underlying array may be allocated in read-only memory. 

prvalue:
> A prvalue ("pure rvalue") expression is an expression that does not have identity and can be moved from. 

So yes, it will be moved if it can or is feasible, typically, but initializer_lists can largely be considered a compile-time construct so my bet is that most of the time the constructor is only invoked statically.